providers/src/airflow/providers/openlineage/plugins/adapter.py

```diff
@@ -128,11 +128,11 @@ def build_task_instance_run_id(
         dag_id: str,
         task_id: str,
         try_number: int,
-        execution_date: datetime,
+        logical_date: datetime,
```

Comment:
I did a quick search and at least `providers/src/airflow/providers/dbt/cloud/utils/openlineage.py` calls this function, so we need additional compatibility measures for different provider version combinations.

It’s probably best to leave this function as-is for now (don’t change the `execution_date` argument in this PR) and figure out the considerations later.

Reply:
Noted